src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -27,12 +27,18 @@ public class LogicManager implements Logic {
     public static final String FILE_OPS_PERMISSION_ERROR_FORMAT =
             "Could not save data to file %s due to insufficient permissions to write to the file or the folder.";
 
+    public static final String MESSAGE_COMMAND_CANCELLED = "Command has been cancelled.";
```

Comment:
Perhaps this string can be located in `Messages.java` for better standardisation